commands/builds/create.js

```diff
@@ -98,6 +97,7 @@ module.exports = {
   description: 'create build',
   flags: [
     { name: 'source-url', description: 'source URL that points to the tarball of your application\'s source code', hasValue: true },
+    { name: 'cwd', description: 'the local path to build. Defaults to pwd', hasValue: true },
```

Comment:
I would change this flag to `dir` or `wd`. `cwd` means "current working directory", you would be specifying it so that it is _different_ than the cwd. `pwd` however means "print working directory", this one should be changed to `cwd`.